### PR TITLE
Haml for clean view code

### PR DESCRIPTION
**Pull request by triszt4n**
### What's new

This PR will implement haml, will change .erb files to .haml files. This new template engine uses a simple indentation based structure for building html files.

### Instructions

For a short instructional post, see [this page](https://medium.com/@admatbandara/why-haml-with-rails-7f67fe1af85a).

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -52,3 +52,5 @@ end
 
 # Windows does not include zoneinfo files, so bundle the tzinfo-data gem
 gem 'tzinfo-data', platforms: [:mingw, :mswin, :x64_mingw, :jruby]
+# Using HAML instead of ERB for transparent view files
+gem "haml", "~> 5.1"
```
Comment (SepsiLaszlo) [inline Gemfile:56]: We could use HAML as the default template engine for scaffolding. No configuration is needed, we just need to add it here:
```suggestion
gem "haml", "~> 5.1"
# Use HAML as the default template engine for scaffolding
gem 'haml-rails', '~> 2.0', '>= 2.0.1'
```

**Review by SepsiLaszlo — CHANGES_REQUESTED** [review]
Good job! The views are a lot nicer!

I mentioned one minor improvement in the review, please take a look.

**Review by bbucsy — APPROVED** [review]
Looks less like a PHP project. I approve!

**Review by SepsiLaszlo — APPROVED** [review]
Great!

**Pull request merged**